Makefile

```diff
@@ -68,7 +68,7 @@ oci-image-tool:
 
 schema-fs:
 	@echo "generating schema fs"
-	@cd schema && echo -e "$$(cat ../.header)\n\n$$(go generate)" > fs.go
+	@cd schema && echo "$$(cat ../.header)\n\n$$(go generate)" > fs.go
```

Comment:
This is not a drive by change. This make target is broken with `-e` and removing it did not change the output.